distributed/diagnostics/tests/test_cudf_diagnostics.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import pytest
+
+from distributed.utils_test import gen_cluster
+
+pytestmark = pytest.mark.gpu
+
+cudf = pytest.importorskip("cudf")
+dask_cuda = pytest.importorskip("dask_cuda")
+
+
+def force_spill():
+    from cudf.core.buffer.spill_manager import get_global_manager
+
+    manager = get_global_manager()
+
+    # 24 bytes
+    df = cudf.DataFrame({"a": [1, 2, 3]})
+
+    return manager.spill_to_device_limit(1)
+
+
+@gen_cluster(
+    client=True,
+    nthreads=[("127.0.0.1", 1)],
+    Worker=dask_cuda.CUDAWorker,
```

Comment:
cc @pentschev this was the impetus for https://github.com/rapidsai/dask-build-environment/pull/73, as using the default worker class here we seem to be unable to initialize the spilling monitoring? This was unexpected to me as I would think all we require is a spilling-enabled installation of cuDF on the workers

Reply:
Would my https://github.com/dask/distributed/pull/8148#discussion_r1340991042 above help with the initialization issues? Otherwise, let's leave it like this for now and discuss how to improve our testing strategy next week.